contrib/hbwin/hbwin.hbp

```diff
@@ -29,7 +29,7 @@ mapi.c
 oemansi.c
 olecore.c
 oleuuid.c{watcom}
-hbolesrv.c
+{HBWIN_HAS_OLESRV}hbolesrv.c
```

Comment:
Ok, probably a fork of your master + my libs to review will be good.
No problem code will be around for a while.
Thanks for help, the _dll suffix shed a light, Harbour bin folder is actually filled with those, I can see the logic used in contrib\make.hb